ETL/ETL/_bezier.h

```diff
@@ -323,7 +323,7 @@ class bezier_base<double,float> : public std::unary_function<float,double>
 	value_type _coeff[4];
 	time_type drs; // reciprocal of (s-r)
 public:
-	bezier_base():r(0.0),s(1.0),drs(1.0) { }
+	bezier_base(): a{0.l}, b{0.l}, c{0.l}, d{0.l}, r(0.0),s(1.0),drs(1.0) { }
```

Comment:
Why aren't these `0.f` as previous case?

Reply:
Just to show they are double. Your question confirms that this is not obvious.

Previous class:
> typedef float value_type;

Current class:
> typedef double value_type;

Reply:
I supposed 1.0f that means is float :)